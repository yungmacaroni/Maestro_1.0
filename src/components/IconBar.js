import React from "react";
import { View, Alert, Pressable } from "react-native";
import {
  MaterialCommunityIcons,
  Foundation,
  MaterialIcons,
} from "@expo/vector-icons";
import { styles } from "./Styles";

export default function IconBar() {
  function onPress() {
    Alert.alert("This feature is not available yet");
  }
  return (
    <View style={styles.iconBar}>
      <Pressable onPress={onPress}>
        <Foundation name="home" size={40} style={styles.homeIcon} />
      </Pressable>
      <Pressable onPress={onPress}>
        <MaterialIcons name="groups" size={40} style={styles.groupsIcon} />
      </Pressable>
      <Pressable onPress={onPress}>
        <MaterialCommunityIcons
          name="account"
          size={40}
          style={styles.profileIcon}
        />
      </Pressable>
    </View>
  );
}
