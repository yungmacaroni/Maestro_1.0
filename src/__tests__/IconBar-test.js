import React from 'react';
import renderer from 'react-test-renderer';
import IconBar from '../components/IconBar';

test('renders correctly', () => {
  const tree = renderer.create(<IconBar />).toJSON();
  expect(tree).toMatchSnapshot();
});