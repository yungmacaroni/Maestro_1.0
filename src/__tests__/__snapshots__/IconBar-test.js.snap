// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#1b411f",
      "flexDirection": "row",
      "height": "8%",
      "justifyContent": "space-evenly",
      "paddingHorizontal": 8,
    }
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <Text />
  </View>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <Text />
  </View>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <Text />
  </View>
</View>
`;
